tensorboard/webapp/metrics/store/metrics_store_internal_utils.ts

```diff
@@ -404,4 +405,28 @@ export function getImageCardStepValues(
   return loadable.runToSeries[runId].map((stepDatum) => stepDatum.step);
 }
 
-export const TEST_ONLY = {getImageCardStepValues, util};
+/**
+ * Returns selected steps that are within selected time given a list of step
+ */
+function getSelectedSteps(
+  selectedTime: LinkedTime | null,
+  stepValues: number[]
```

Comment:
Sorry, this request is a bit belated: But consider not using the term "stepValues" since the "values" doesn't really mean anything  to me - they are just "steps" and there are no additional "values", right?

Same advice for getImageCardStepValues. Maybe just call it getImageCardSteps.

Reply:
Sure! let me make rename `getImageCardSteps` a separate PR